notebooks/scripts/plot_effect_estimation.py

```
@@ -42,30 +42,59 @@
+    x_var="effect_counterfactual",
+    x_label="True ATE",
+    filename="effect_estimation_different_effect_strength_a.png",
+    xticks=[-0.25, 0, 0.1, 0.2, 0.3, 0.4, 0.5],
+    yticks=[-0.3, -0.2, -0.1, 0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7],
+    plot_baseline=False,
+)
+
+
+ax.plot([-0.25, 0.4], [-0.25, 0.4], color="#696969", linestyle="--", label="True ATE")
+ax.legend(ncol=4, frameon=False, bbox_to_anchor=(0.5, 1.1), loc="upper center")
+fig
```

Ruff

Found useless expression. Either assign it to a variable or remove it.

(B018)